src/org/sparkboard/server/server.clj

```diff
@@ -523,13 +390,213 @@
     (case action
       :link-account! (link-account! context params))))
 
+(defn action-value [action]
```

Comment:
What's the expected data model of an incoming `action`?

Reply:
I'm thinking that might make sense in a docstring or `:pre` condition

Reply:
https://api.slack.com/reference/interaction-payloads/block-actions has some info. I haven't  found any good reference and am just expanding this function as I use more types.

> I'm thinking that might make sense in a docstring or :pre condition
What is this referring to? GH isn't showing me the context of this comment

Reply:
I meant a docstring/`:pre` condition to document the shape/keys of an incoming `action` in this `action-value` function.